docs/docs-beta/docs/dagster-plus/features/authentication-and-access-control/sso/authentication.md

```diff
@@ -4,3 +4,4 @@ sidebar_position: 100
 unlisted: true
 ---
 
+{/* TODO - populate */}
```

Comment:
Is this new content?

Reply:
I think you can delete this file -- it was around when I took over the docs revamp, but I can't find any useful context for it.